pkg/proxy/accesslog/log.go

```diff
@@ -134,3 +204,102 @@ func Log(l *LogRecord, typ FlowType, verdict FlowVerdict, code int) {
 		log.WithError(err).WithField(FieldFilePath, logPath).Warn("Error encountered while flushing to access log")
 	}
 }
+
+// fillIdentity resolves the labels of the specified identity if known
+// locally and fills in the following info member fields:
+//  - info.Identity
+//  - info.Labels
+//  - info.LabelsSHA256
+func (info *EndpointInfo) fillIdentity(id policy.NumericIdentity) {
+	info.Identity = uint64(id)
+
+	if c := policy.GetConsumableCache().Lookup(id); c != nil {
+		if c.Labels != nil {
+			info.Labels = c.Labels.Labels.GetModel()
+			info.LabelsSHA256 = c.Labels.GetLabelsSHA256()
+		}
+	}
+}
+
+// egressDestinationInfo returns the destination EndpointInfo for a flow
+// leaving the proxy at egress.
+func (info *EndpointInfo) egressDestinationInfo(ipstr string) {
+	ip := net.ParseIP(ipstr)
+	if ip != nil {
```

Comment:
This could be `if ip == nil { return }` if you wanted to reduce the indent by one.

Reply:
OK, thanks!